src/Microsoft.TemplateEngine.Edge/TemplateUpdates/InstallUnitDescriptorFactory.cs

```diff
@@ -53,14 +85,27 @@ public static bool TryParse(IEngineEnvironmentSettings environmentSettings, JObj
         // For creating descriptors.
         public static bool TryCreateFromMountPoint(IEngineEnvironmentSettings environmentSettings, IMountPoint mountPoint, out IReadOnlyList<IInstallUnitDescriptor> descriptorList)
         {
+            IInstallUnitDescriptorFactory defaultFactory = null;
+
             foreach (IInstallUnitDescriptorFactory factory in environmentSettings.SettingsLoader.Components.OfType<IInstallUnitDescriptorFactory>().ToList())
             {
+                if (factory is DefaultInstallUnitDescriptorFactory)
+                {
+                    defaultFactory = factory;
+                    continue;
+                }
+
                 if (factory.TryCreateFromMountPoint(mountPoint, out descriptorList))
                 {
                     return true;
                 }
             }
 
+            if (defaultFactory.TryCreateFromMountPoint(mountPoint, out descriptorList))
```

Comment:
We can probably just do `return defaultFactory.TryCreateFromMountPoint(mountPoint, out descriptorList);` here, as it returns true if true/false if false and `descriptorList` should be discarded on false return (unless we're guaranteeing that it's null on false return)